### PR TITLE
Update release notes for 1.6.0

### PR DESCRIPTION
**Pull request by fujunwei**

### CODE REVIEW

**Comment by fujunwei** [comment]
@baleboy @lincsoon @wuhengzhi PTAL.

File: README.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ You can also set user agent with the preference of xwalkUserAgent.
 
 ### Release Notes
 
+#### 1.6.0 (March 11, 2016)
+* Uses the latest Crosswalk 17 stable version by default
+* Support to [package apps for 64-bit devices](https://crosswalk-project.org/documentation/android/android_64bit.html), It's possible to specify 64-bit targets using the --xwalk64bit option in the build command:
```
Comment (axinging) [inline README.md]: It's=>it's

**Comment by fujunwei** [comment]
Thanks, Done.

**Comment by axinging** [comment]
LGTM

**Pull request merged**